package.json

```diff
@@ -22,8 +22,8 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "chart.js": "^2.8.0",
-    "chartjs-node": "^1.7.1",
+    "chart.js": "^2.9.4",
+    "chartjs-node-canvas": "^3.2.0",
```

Comment:
the new legend text is extremely hard to read, could we fix the font color there?

Reply:
Ah that is correct, let me check.